build.sbt

```diff
@@ -611,12 +600,8 @@ lazy val `scio-google-cloud-platform`: Project = project
   .settings(
     description := "Scio add-on for Google Cloud Platform",
     libraryDependencies ++= Seq(
-      "com.google.cloud" % "google-cloud-spanner" % googleCloudSpannerVersion excludeAll (
-        ExclusionRule(organization = "io.grpc")
-      ),
-      "com.google.cloud.bigtable" % "bigtable-client-core" % bigtableClientVersion excludeAll (
-        ExclusionRule(organization = "io.grpc")
-      ),
```

Comment:
Conflicts are handled by global `dependencyOverrides`. It would be preferable to see conflict in evicted rather that dropping a required transitive grpc dep